third_party/tsl/.bazelrc

```diff
@@ -234,22 +237,22 @@ build:cuda_clang --@local_config_cuda//:cuda_compiler=clang
 # release while SASS is only forward compatible inside the current
 # major release. Example: sm_80 kernels can run on sm_89 GPUs but
 # not on sm_90 GPUs. compute_80 kernels though can also run on sm_90 GPUs.
-build:cuda_clang --repo_env=TF_CUDA_COMPUTE_CAPABILITIES="sm_60,sm_70,sm_80,sm_89,compute_90"
+build:cuda_clang --repo_env=HERMETIC_CUDA_COMPUTE_CAPABILITIES="sm_60,sm_70,sm_80,sm_89,compute_90"
```

Comment:
`compute_90` really should be `sm_90,compute_90` -- but I vaguely recall that the crosstool wrapper automatically expands that; is that correct?

Reply:
I haven't changed the capabilities list, just the name of the environment variable.
As discussed during our meeting, your assumption is correct.